src/TheCallCenter/TheCallCenter.csproj

```diff
@@ -1,14 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Need <NullableReferenceTypes>true</NullableReferenceTypes>

Reply:
what specifically needs true? I have added <Nullable>enable</Nullable> and <LangVersion>10.0</LangVersion> in my local branch (not yet updated) but none of those use "true". Also, <AspNetCoreHostingModel> seems to only be InProcess or OutOfProcess?